packages/kit-bg/src/vaults/impls/evm/sdkEvm/BaseApiProvider.ts

```
@@ -2,10 +2,11 @@
 
 import BigNumber from 'bignumber.js';
 import { md5 } from 'js-md5';
-import { isEmpty, isNaN, keyBy, omit, orderBy, uniqBy } from 'lodash';
+import { forEach, isEmpty, isNaN, keyBy, omit, orderBy, uniqBy } from 'lodash';
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)